### ISSUE TITLE
publish-iis on 1.1.0 runtime

### ISSUE THREAD

**Issue by JunTaoLuo** [aspnet/IISIntegration#289]
Since we didn't anticipate any changes to the dotnet tool publish-iis, we have removed it from this repo and do not plan to ship any newer package in the future. However, when upgrading web applications to netcoreapp1.1 with all 1.1.0 dependencies, restoring for the publish-iis tool and trying to use it will fail and output the message.
```
The specified framework 'Microsoft.NETCore.App', version '1.0.0' was not found.
  - Check application dependencies and target a framework version installed at:
      /usr/local/share/dotnet/shared/Microsoft.NETCore.App
  - The following versions are installed:
      1.0.0-rc2-23925
      1.1.0-preview1-001100-00
  - Alternatively, install the framework version '1.0.0'.
```

 This is because the publish-iis tool itself has a dependency on Microsoft.NetCore.App version 1.0.0:
```
"Microsoft.AspNetCore.Server.IISIntegration.Tools/1.0.0-preview2-final": {
        "type": "package",
        "dependencies": {
          "Microsoft.DotNet.ProjectModel": "1.0.0-rc3-003121",
          "Microsoft.Extensions.CommandLineUtils": "1.0.0",
          "Microsoft.NETCore.App": "1.0.0",
          "System.Diagnostics.Process": "4.1.0"
        },
        "compile": {
          "lib/netcoreapp1.0/dotnet-publish-iis.dll": {}
        },
        "runtime": {
          "lib/netcoreapp1.0/dotnet-publish-iis.dll": {}
        }
      }
```

cc @natemcmaster @muratg @glennc

**Comment by muratg** [comment]
@natemcmaster thoughts?

**Comment by natemcmaster** [comment]
We should releasing a version for the 1.1 runtime. The people who will hit this are people trying .NET Core for the first time. There is like a 0% chance they'll figure out how to [install the 1.0.0 shared framework.](https://github.com/dotnet/cli/blob/7da6ee032572b796e2b0d9c6a89072fdf6e060fd/scripts/obtain/dotnet-install.ps1#L32-L34)

**Comment by natemcmaster** [comment]
Resolved in #290.
